src/Visualization/Python/PlotAlongLine.py

```diff
@@ -52,7 +62,7 @@ def _parse_point(ctx, param, value):
 )
 @click.option(
     "--step",
-    type=int,
+    callback=_parse_step,
     help="Observation step number. Specify '-1' for the last step in the file.",
 )
```

Comment:
So that it's consistent with `Clip`?
```py
@click.option(
    "--time-step",
    "-t",
    callback=_parse_step,
    help=(
        "Select a time step. Specify '-1' or 'last' to select the last time"
        " step. You can also specify 'first' for the first time step."
    ),
)
```

Reply:
Currently it's consistent with `PlotPowerMonitors`. Seems like we should make the `render-3d` commands consistent with the `plot` commands.